packages/types/message.d.ts

```diff
@@ -16,9 +16,13 @@ export type CumulusMessage = {
   cumulus_meta: CumulusMeta
   meta: object
   payload: unknown
+  exception?: object
```

Comment:
Do we know what properties an `exception` has?

Reply:
actually it's not even always an object. It can be "None" if there is no exception, or it can be an object and I'd have to look at the properties. But I guess it should be "unknown" given the uncertainty?

Reply:
Let's make it `'None' | object` for now, and we can refine it later.